docker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export NAME=osr_django
+export PORTS="-p 8000:8000"
+export VOLUME="-v $(pwd):/app"
+
+docker stop $NAME
+sleep 1
+docker rm $NAME
+sleep 1
+docker build -t osr/$NAME . || exit 1
```

Comment:
same re `set -eu`

Reply:
No actually because the docker rm might fail the first time, so if I set -eu here we'll have to do || true at the end of those two lines (the rm and the stop)